app/src/main/java/com/openclassrooms/realestatemanager/ui/EstateListAdapter.kt

```diff
@@ -28,9 +36,19 @@ class EstateListAdapter(
         holder.type.text = getEstateTypeString(estate.type)
         holder.city.text = estate.city
         holder.price.text = decimalFormat.format(estate.price)
+        holder.itemView.setOnClickListener { estateListAdapterListener.onEstateItemClick(estate) }
+        if (selectedEstate == estate) {
+            holder.itemView.setBackgroundColor(ResourcesCompat.getColor(resources, R.color.estate_list_selected_estate, null))
+            holder.price.setHintTextColor(ResourcesCompat.getColor(resources, R.color.estate_list_selected_estate_text, null))
+        }
+        if (lastSelectedEstate == estate) {
```

Comment:
```suggestion
        else {
```

Si ça te permet de ne pas garder "lastSelectedEstate" en mémoire

Reply:
Mais du coup on rechange la couleur de tous les élements de la liste, cela fait plus de calcul que de garder une valeur en mémoire.